berkeley-mobile/StudyPact/ProfileViewController.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  ProfileViewController.swift
+//  berkeley-mobile
+//
+//  Created by Oscar Bjorkman on 1/19/21.
+//  Copyright © 2021 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController, UITextFieldDelegate {
+    
+    private var loggedIn = false
+    
+    private var profileLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.view.layoutMargins = UIEdgeInsets(top: 16, left: 16, bottom: 16, right: 16)
+        self.view.backgroundColor = Color.modalBackground
+        
+        setupHeader()
+        setupProfile()
+        
+        let tap = UITapGestureRecognizer(target: self, action: #selector(UIInputViewController.dismissKeyboard))
+        view.addGestureRecognizer(tap)
+    }
+    
+    @objc func dismissKeyboard() {
+        view.endEditing(true)
+    }
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        // TODO: - Update profile info here
+        view.endEditing(true)
+        return false
+    }
+}
+
+extension ProfileViewController {
+    func setupHeader() {
+        profileLabel = UILabel()
+        profileLabel.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        profileLabel.numberOfLines = 0
+        profileLabel.font = Font.bold(30)
+        profileLabel.text = "Profile"
+        view.addSubview(profileLabel)
+        profileLabel.translatesAutoresizingMaskIntoConstraints = false
+        profileLabel.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 15).isActive = true
+        profileLabel.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
+        
+        // Blob
+        let blob = UIImage(named: "BlobTopRight")!
+        let aspectRatio = blob.size.width / blob.size.height
+        let blobView = UIImageView(image: blob)
+        blobView.contentMode = .scaleAspectFit
+
+        view.addSubview(blobView)
+        blobView.translatesAutoresizingMaskIntoConstraints = false
+        blobView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1 / 1.3).isActive = true
+        blobView.heightAnchor.constraint(equalTo: blobView.widthAnchor, multiplier: 1 / aspectRatio).isActive = true
+        blobView.topAnchor.constraint(equalTo: view.topAnchor, constant: -blobView.frame.height / 3.5).isActive = true
+        blobView.rightAnchor.constraint(equalTo: view.rightAnchor, constant: 40).isActive = true
```

Comment:
this blob code should probably be unified somewhere (maybe a class thats like BlobBackgroundView) because it's used multiple places and we will likely use it on future pages.

Reply:
I'll leave this for another PR